src/mocks/mockISelectionManager.ts

```diff
@@ -112,6 +133,8 @@ export class MockISelectionManager implements ISelectionManager {
     }
 
     public simutateSelection(selections: ISelectionId[]): void {
-        this.callback(selections);
+        if (this.callback && typeof this.callback === "function") {
```

Comment:
type guard is enough here, it also checks for undefined

Reply:
Fixed.